### PR TITLE
merge tensorboard-integration-ap with tensorboard-integration-ku

### PR DESCRIPTION
**Pull request by anyap0p**

### CODE REVIEW

File: train/train_clf.py [hunk 1]
```diff
@@ -24,6 +24,11 @@
 import torch
 import numpy as np
 
+import torch
+import torchvision
+from torch.utils.tensorboard import SummaryWriter
+from torchvision import datasets, transforms
```
Comment (kadhirumasankar) [inline train/train_clf.py]: We don't need these. You can delete them

File: train/train_clf.py [hunk 2]
```diff
@@ -230,6 +235,11 @@ def train(
             log.metric("EMA score", ema_score, epoch)
             log.metric("Best EMA score", scores["best_ema_score"], epoch)
 
+            log.metric("new model highest score", new_model_highest_score, epoch)
+            log.metric("model highest score", scores["model_highest_score"], epoch)
+            log.metric("new ema highest score", new_ema_highest_score, epoch)
+            log.metric("ema highest score", scores["ema_highest_score"], epoch)
+
```
Comment (kadhirumasankar) [inline train/train_clf.py]: You can delete these

**Review by kadhirumasankar — CHANGES_REQUESTED** [review]
Just need to delete a few lines and I'll merge your branch with mine

**Review by kadhirumasankar — APPROVED** [review]
Looks good to me 👍

File: train/train_utils/logger.py [hunk 3]
```diff
@@ -26,7 +26,8 @@ def __init__(self, log_file: pathlib.Path) -> None:
         console.setFormatter(formatter)
         # add the handler to the root logger
         logging.getLogger("").addHandler(console)
-        self.tensorboard_writer = SummaryWriter(log_dir=log_file.parent)
+        # to write metrics to tensorboard
+        self.writer = SummaryWriter(log_dir=log_file.parent)
```
Comment (alexwitt23) [inline train/train_utils/logger.py:30]: See style guide [here](https://google.github.io/styleguide/pyguide.html#22-imports) on importing modules.

**Review by alexwitt23 — APPROVED** [review]
lgtm

**Pull request merged**